src/main/resources/application.yml

```diff
@@ -57,8 +61,8 @@ spring:
     defer-datasource-initialization: true
 
     database-platform: org.hibernate.dialect.H2Dialect
-    #database-platform: org.hibernate.dialect.PostgreSQLDialect
-    #database-platform: org.hibernate.dialect.MySQL5InnoDBDialect
+    # database-platform: org.hibernate.dialect.PostgreSQL95Dialect
```

Comment:
Postgresql 9 itself is being deprecated by hibernate.
- https://hibernate.atlassian.net/browse/HHH-15487

We should add a comment about `PostgreSQL95Dialect` and similar being used for the appropriate 9.x version but the `PostgreSQLDialect` requires a postgresql of at least 10.x.

Reply:
Added comment. Thanks @kaladay